### PR TITLE
Use volume mount in Dockerfile.dev

### PR DESCRIPTION
**Pull request by kei-s**

### CODE REVIEW

**Comment by 284km** [comment]
Thanks!

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ e.g.
 
 ```
 $ docker build -f ./Dockerfile.dev -t charty-dev:0.1 .
-$ docker run -it charty-dev:0.1 irb -rcharty
+$ docker run -it -v ${PWD}:/charty -w /charty charty-dev:0.1 ./bin/console
```
Comment (mrkn) [inline README.md]: I think we sometimes need to run `bundle install` before running `bin/console`.

Comment (284km) [reply README.md]: You are quite right.  I will add `$ bundle install` to the head.

Comment (284km) [reply README.md]: Fixed it. https://github.com/red-data-tools/charty/commit/b57b0aedf12c427f80191f15777b5586e1a99ea9

**Pull request merged**